src/plugins/data/public/ui/search_bar/search_bar.tsx

```diff
@@ -28,27 +28,25 @@
  * under the License.
  */
 
-import { compact } from 'lodash';
 import { InjectedIntl, injectI18n } from '@osd/i18n/react';
 import classNames from 'classnames';
+import { compact, get, isEqual } from 'lodash';
 import React, { Component } from 'react';
 import ResizeObserver from 'resize-observer-polyfill';
-import { get, isEqual } from 'lodash';
```

Comment:
nit: quick question. was this auto sorted? should be fine but something to watch out incase some folks working on the discover-next stuff have their own linting pref as well to avoid bigger deltas in the changes.

if this was we could also consider adding the rule here: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.eslintrc.js

Reply:
yes they were auto sorted. but this came from the typescript language server, i'm not sure if there's a corresponding eslint rule. i agree it would be better to enforce a consistent rule on import order